src/extension.ts

```diff
@@ -110,7 +110,14 @@ const createHelperExecutable = (
 
 const authenticateAdo = async (context: vscode.ExtensionContext) => {
   try {
-    await getAccessToken();
+    var scopes = ["499b84ac-1321-427f-aa17-267ca6975798/.default"];
```

Comment:
I would love it if we could move this to a Constant variable declared somewhere at the top of this file. Something like `const DEFAULT_ADO_SCOPE = "499b84ac-1321-427f-aa17-267ca6975798/.default";`

Reply:
Done in new commit